ossfuzz/fuzz_data_producer.c

```diff
@@ -0,0 +1,32 @@
+#include "fuzz_data_producer.h"
+
```

Comment:
Let's hide the definition of the struct. 

```c
struct FUZZ_dataProducer_s {
  const uint8_t *data;
  size_t size;
};
```

Then where you are using `producer->size`, instead add a function `size_t FUZZ_dataProducer_remainingBytes(FUZZ_dataProducer_t* producer) { return producer->size; }`